### DISCUSSION TITLE
PVE Leaderboards

### DISCUSSION THREAD

**Discussion by Hyperbearr** [chromiecraft/chromiecraft#3710] [Suggestions]
Hi, 

So since it came up in conversation in the BWL Channel on the discord and we've had pvp rewards for what feels like an eternity, I thought I'd outline some of the suggestions regarding potential PVE leaderboards that felt reasonable. 

**All Star Rankings**
- Points provided for various standings (to follow) that would mirror that used by Warcraftlogs or similar
- Provides a comparable metric to the PVP leaderboard which can be used to provide similar rewards
- Reset on a seasonal basis as new progression patches drop (New raids and new level brackets)

**Dungeon Rankings**
- Timer starts when RDF queue pops
- Timer ends when the RDF boss requirements are met
- Reset on a monthly basis to provide continued incentives to compete
- Have a legacy leaderboard of all fastest runs by season and ever to capture runs not included in the current month
- Provide all star points for the best runs

**Raid Rankings**
Primarily we discussed that boss kill times feels like the most reasonable metric that would see the most player engagement with these boss kill time ranks contributing all star points. It also minimises drama and effort for the devs as you don't have to worry about appeasing everyone with 'sensible' trash requirements, start and end points or any other weird shenanigans that come with full raid speedruns. 

All of this has been done on other WOTLK servers so happy to put forth examples of best practice and implementation that we'd love to see replicated. Equally would love others to chime in with thoughts on the above. 

Best,
A PVE Andy

**Comment by RestoCC** [comment]
A few guilds on CC have been doing dungeon speedrun events. We've seen challenges like Stratholme Undead, UBRS and now Scholomance. Below is the most recent event and rules created. This may give some ideas for people.


__**Scholomance Speedrun**__

**Prize**
*1st place- Gold Pot of 1000g*
*2nd place- Gold Pot of 500g*
*3rd place- Gold Pot of 250g*

**Due Date**
*- Entries due by the reset of server on July 6 Server Time

**Rules**
*- Must queue through Dungeon Finder*
*- Any specs/comp is allowed*
*- Timer starts as you zone in*
*- Must kill the following:*
      **- Jandice Barov**
      **- Marduk Blackpool & Vectus**
       **- Darkmaster Gandling**
*- No Trash requirements*
*- Only one submission per group*
*- Each character (not person) is locked to their one group*
***- Run must be filmed and shared to X in private DM before due date, at the end Ill link the top 3 video runs in the channel. This way there's no leeching strategies off other players***

**Prize**
*- Gold Pot of 1750g*

**Comment by KaneoHunter** [comment]
![anime-excited](https://user-images.githubusercontent.com/15280301/176689555-8f39ae63-2a6b-4b48-8061-e9558a50c06c.gif)

**Comment by Arbit0** [comment]
Although I'd love to see something of this sort what I'm concerned about is much of a role raid composition would play a role in this, similar to how Warcraft logs parses would sometimes be unattainable without an exact replica comp.

I'm much too stupid to have any suggestion on how to work around this to be more incentivizing to the less hardcore-oriented guilds.

However if this is just an implementation similar to how the PvP rewards system works but for PvE players I'd be ecstatic about this, as the PvP rewards/leaderboard etc. also wholly caters to the PvP focused community on the server.

**Reply by Dupeloop** [reply]
Why not just play on a competitive pve server that has a lot more serious players. The competition would be better since you're not just competing with CC casuals. CC also isn't a try hard server why do we need any leaderboards that promote sweaty competition.

**Reply by Hyperbearr** [reply]
There's a PVP leaderboard, why not a PVE leaderboard? 

What tryhard and sweaty PTE server? There aren't any. 

Also, many of us are still committed to testing and improving AC - as evidenced by continued engagement with bug reports, developers, PTR testing etc etc - but are looking for the addition of tools and changes that will allow us to have more fun whilst doing that.
